website/homepage/src/docs/template.html

```diff
@@ -21,6 +21,10 @@
         {% for component in docs["component"] %}
           <a class="px-4 block thin-link" href="#{{ component['name'].lower() }}">{{ component['name'] }}</a>
         {% endfor %}
+        <a class="link px-4 my-2 block" href="#components-helpers">Component Helpers
```

Comment:
This links to `#components-helpers` but the actual section has `id="component-helpers"` (no s), so currently nothing happens when you click.